config/webpack.config.dev.js

```diff
@@ -142,7 +142,7 @@ module.exports = {
           },
           {
             test: /\.(js|jsx|mjs)$/,
-            include: paths.appSrc,
+            include: [paths.appSrc, paths.appNodeModules +  '/@polkadot/util/node_modules/chalk/source/'],
```

Comment:
Are we adding this because of a mis-config from this polkadot package?

Reply:
yes, this package is using the object-spread feature, which has to be transpiled, to be usable